calico_containers/pycalico/block.py

```diff
@@ -214,10 +237,12 @@ def auto_assign(self, num, handle_id, attributes, host,
                 ips.append(ip)
         return ips
 
-    def assign(self, address, handle_id, attributes):
+    def assign(self, address, handle_id, attributes, host):
```

Comment:
Yep, sounds fine to me.

I was just having similar thoughts myself and was about to delete the comment :)